components/lv_examples/component.mk

```diff
@@ -4,24 +4,12 @@
 
 CFLAGS += -DLV_CONF_INCLUDE_SIMPLE
 
-COMPONENT_SRCDIRS := lv_examples      \
-    lv_examples/lv_apps/benchmark         \
-    lv_examples/lv_apps/demo              \
-    lv_examples/lv_apps/sysmon            \
-    lv_examples/lv_apps/terminal          \
-    lv_examples/lv_apps/tpcal             \
-    lv_examples/lv_tests/lv_test_theme     \
-    lv_examples/lv_tests/lv_test_group     \
-    lv_examples/lv_tutorial/10_keyboard   \
-    lv_examples/lv_tutorial/1_hello_world \
-    lv_examples/lv_tutorial/2_objects     \
-    lv_examples/lv_tutorial/3_styles      \
-    lv_examples/lv_tutorial/4_themes      \
-    lv_examples/lv_tutorial/5_antialiasing  \
-    lv_examples/lv_tutorial/6_images        \
-    lv_examples/lv_tutorial/7_fonts         \
-    lv_examples/lv_tutorial/8_animations    \
-    lv_examples/lv_tutorial/9_responsive
-
+COMPONENT_SRCDIRS := lv_examples           \
+    lv_examples/src/lv_demo_benchmark      \
+    lv_examples/src/lv_demo_keypad_encoder \
+    lv_examples/src/demo_stress            \
+    lv_examples/src/lv_demo_widgets        \
+    lv_examples/src/lv_ex_style            \
+    lv_examples/src/lv_ex_widgets
```

Comment:
[lv_examples/assets](https://github.com/lvgl/lv_examples/tree/master/assets) might be also required.